composer.json

```diff
@@ -10,23 +10,25 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0",
```

Comment:
Probably doesn't matter tremendously since it's a pretty open constraint, but why remove it?

Reply:
An oversight on that. I had intended to bump the minimum to 7.2. See #20. But then changed scope, close that PR and opened this one. Now to clean things up. New pushes inbound.